.github/workflows/auto-commit-client-updates.yml

```diff
@@ -1,61 +0,0 @@
-# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
```

Comment:
Why is this workflow being removed? Please provide justification for changes. I don't have enough context to understand the changes you're making. Either provide smaller separate PRs or provide a lot more context to me so that I can understand the changes you're making.

Reply:
it was replaced by new flow to run tests for commit which will provide the code is compilible and prove that tests are good